packages/rum/test/viewTracker.spec.ts

```diff
@@ -214,5 +216,30 @@ describe('rum view measures', () => {
       resourceCount: 0,
       userActionCount: 0,
     })
+    restorePageVisibility()
+  })
+  it('should not collect firstContentfulPaint if page is not visible', () => {
+    setPageVisibility('hidden')
+    const lifeCycle = new LifeCycle()
+    setup({ addRumEvent, lifeCycle })
+    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, FAKE_PAINT_ENTRY as PerformancePaintTiming)
+    lifeCycle.notify(
+      LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED,
+      FAKE_NAVIGATION_ENTRY as PerformanceNavigationTiming
+    )
+    history.pushState({}, '', '/bar')
+
+    expect(getEventCount()).toEqual(3)
+    expect(getViewEvent(1).view.measures).toEqual({
```

Comment:
```suggestion
    expect(getViewEvent(1).view.measures. firstContentfulPaint).toBeUndefined()
```
it would make clearer what we expect